lib/management/push.js

```diff
@@ -261,8 +272,30 @@ function addOverlayMessageListener(callback) {
     overlayMessageListener = callback;
 }
 
+function startPushConnectionHealthCheckServer(cb) {
+    const server = http.createServer((req, res) => {
+        const path = parse(req.url).pathname;
```

Comment:
I think `URL.parse` is considered legacy code (https://nodejs.org/docs/latest-v10.x/api/url.html#url_url_strings_and_url_objects), for being future-proof it might be better to use `new URL` instead and access the object attributes.

Reply:
So in classic nodejs fashion, the `url` member is not actually a valid URL but just a path (`new URL` on it throws), and I guess that's the reason the 10.x `http` module docs still use `parse`. I'll just use `req.url` directly with no parsing since it's for healthchecks that won't be adding any hashes or a query string.